From 4704c84825474ebe2f1291b9158087945f88063d Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Fri, 24 Apr 2020 09:09:52 -0500
Subject: [PATCH 12/12] beacon_imx8mm: Enable GPIO Expanders

There are a few GPIO expanders.  One of them controls the
reset line to the USB host hub on the baseboard.  Without
this, the USB hub stays in reset and it cannot communicate
with devices on it.

This patch enables the GPIO expander driver to allow the
the hub to come out of reset.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 configs/beacon_imx8mm_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/beacon_imx8mm_defconfig b/configs/beacon_imx8mm_defconfig
index 852d48ff8a..ca250c1bda 100644
--- a/configs/beacon_imx8mm_defconfig
+++ b/configs/beacon_imx8mm_defconfig
@@ -49,6 +49,7 @@ CONFIG_FASTBOOT_BUF_ADDR=0x42800000
 CONFIG_FASTBOOT_BUF_SIZE=0x40000000
 CONFIG_FASTBOOT_FLASH=y
 CONFIG_DM_GPIO=y
+CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
 CONFIG_SYS_I2C_MXC=y
 CONFIG_SYS_I2C_MXC_I2C1=y
-- 
2.17.1

